files/common/usr/bin/delphix-startup-screen

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/python3
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import curses
+import curses.ascii
+import curses.textpad
+import sys
+import os
+import logging
+import time
+import subprocess
+
+STATUS_HEADER = "STATUS\t\t\tSERVICE\t\t\t"
+LAYOUT_STR = "Keyboard layout (press 9 to change):"
+CONSOLE_STR = "Press any other key for console access."
+KEY_LAYOUT_SELECT_STR = "Select a keyboard layout by number [1-%d]:"
+
+log_file = "/tmp/dpx_startup_screen.log"
+logo = "Delphix Engine Version "
+
+website = "www.delphix.com"
+
+WIN_LEN = 0
+WIN_HT = 0
+
+def get_svcs():
+    cmd = ['systemctl', 'show', '-p',  'Wants', 'delphix.target']
+    cp = subprocess.run(cmd, stdout=subprocess.PIPE, universal_newlines=True)
+    return cp.stdout.split("=", 1)[1].split()
+
+def run_svcs(options=None):
+    '''
+    Run the command to retrieve the status of all the services. The options
+    argument can be used to obtain additional output from the 'systemctl'
+    command. By default we only obtain the 'ActiveState' of the services.
+    '''
+
+    fields = 'ActiveState'
+    cmd = ['systemctl', 'show', '--value', '-p']
+    if options:
+        fields += ',' + options
+    cmd.append(fields)
+    for svc in get_svcs():
+        cmd.append(svc)
+    cp = subprocess.run(cmd, stdout=subprocess.PIPE, universal_newlines=True)
+    return cp.stdout
+
+def get_keyboard_layout():
+    cmd = ['localectl', 'status']
+    cp = subprocess.run(cmd, stdout=subprocess.PIPE, universal_newlines=True)
+    output = cp.stdout.split()
+    return output[output.index("Layout:") + 1]
+
+def set_keyboard_layout(layout):
+    cmd = ['localectl', 'set-x11-keymap', layout, 'pc105']
+    cp = subprocess.run(cmd, shell=False)
+    return subprocess.run('setupcon', shell=False)
+
+def get_valid_keyboard_layouts():
+    cmd = ['localectl', '--no-pager', 'list-x11-keymap-layouts']
+    cp = subprocess.run(cmd, stdout=subprocess.PIPE, universal_newlines=True)
+    return cp.stdout.splitlines()
+
+def load_header(stdscr):
+    global logo
+
+    cmd = ['/opt/delphix/server/bin/get_appliance_version']
```

Comment:
Let's use `get-appliance-version` instead, which should be in the default PATH.

We should probably remove the `get_appliance_version` script that's delivered by the app-gate, to avoid this sort of confusion.

Reply:
We don't want to use `get_appliance_version` since that'd cause the platform package to be dependent on the virtualization package (at least for this script), which we don't want. The `get-appliance-version` script is delivered by the platform package, so there's no package dependency issues this way.